base-server/index.js

```diff
@@ -496,6 +509,16 @@ export class BaseServer {
       channel.regexp = pattern
     }
     this.channels.push(channel)
+
+    if (queue) {
+      channel.regexp
+        ? this.queueTypes.set(channel.regexp, queue)
```

Comment:
Channel could have the same name as some `action.type`. We need a prefix here or a different `Map`.